backend/protos/xyz/block/ftl/timeline/v1/event.proto

```diff
@@ -139,15 +139,19 @@ message Event {
   google.protobuf.Timestamp time_stamp = 1;
   // Unique ID for event.
   int64 id = 2;
+  EventEntry entry = 3;
+}
+
+message EventEntry {
```

Comment:
EventEntry was created was created just so we can reuse the sumtype within `Event` and `CreateEvent`.
The extra level of wrapping is ugly everywhere else though...

Reply:
I suppose I can just repeat the `oneof` in both places and it'd make it a lot nicer to deal with events everywhere (the 99% case)

Reply:
(removed now)